### PR TITLE
translate Container Environment into chinese

### PR DESCRIPTION
**Pull request by fancc**
translate Container Environment into chinese

### CODE REVIEW

File: content/zh/docs/concepts/containers/container-environment.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+```shell
+FOO_SERVICE_HOST=<the host the service is running on>
+FOO_SERVICE_PORT=<the port the service is running on>
+```
+
+<!--
+Services have dedicated IP addresses and are available to the Container via DNS,
+if [DNS addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/) is enabled. 
+-->
+服务具有专用的 IP 地址，如果启用了 [DNS插件](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/)，就可以通过 DNS 用于容器。
```
Comment (sftim) [inline content/zh/docs/concepts/containers/container-environment.md:82]: Should this be:
```suggstion
Service有专用的 IP 地址，如果启用了 [DNS插件](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/)，就可以通过 DNS 用于容器。
```
or similar?

File: content/zh/docs/concepts/containers/container-environment.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+-->
+### 集群信息
+
+创建容器时正在运行的所有服务的列表都可用作该容器的环境变量。这些环境变量与 Docker 链接的语法匹配。
+
+对于名为 *foo* 的服务，它映射到名为 *bar* 的容器，定义了以下变量：
+
+```shell
+FOO_SERVICE_HOST=<the host the service is running on>
+FOO_SERVICE_PORT=<the port the service is running on>
```
Comment (sftim) [inline content/zh/docs/concepts/containers/container-environment.md:75]: It's OK to localize the text inside `<…>`

File: content/zh/docs/concepts/containers/container-environment.md (hunk 1 above)
Comment (fancc) [reply content/zh/docs/concepts/containers/container-environment.md]: I usually call `Service` directly in Chinese '服务' in my work, so it is actually the same. Maybe using English would be better. Thanks!

File: content/zh/docs/concepts/containers/container-environment.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+* A filesystem, which is a combination of an [image](/docs/concepts/containers/images/) and one or more [volumes](/docs/concepts/storage/volumes/).
+* Information about the Container itself.
+* Information about other objects in the cluster.
+-->
+## 容器环境
+
+Kubernetes 的容器环境给容器提供了几个重要的资源：
+
+* 文件系统，包括的一个 [image](/docs/concepts/containers/images/) 和一个或多个的 [volumes](/docs/concepts/storage/volumes/) 的组合。
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/container-environment.md:32]: image -> 镜像
volumes -> 卷

包括的 。。。。 -> 其中包含一个镜像和一个或多个卷。

File: content/zh/docs/concepts/containers/container-environment.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+User defined environment variables from the Pod definition are also available to the Container,
+as are any environment variables specified statically in the Docker image.
+-->
+### 容器信息
+
+容器的 *hostname* 是它所运行在的 pod 的名称。它可以通过 `hostname` 命令或者调用 libc 中的 [`gethostname`](http://man7.org/linux/man-pages/man2/gethostname.2.html) 函数来获取。
+
+Pod 名称和命名空间可以通过 [downward API](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/) 使用环境变量。
+
+Pod定义中的用户所定义环境变量也可用于容器，就像在 Docker 镜像中静态指定的任何环境变量一样。
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/container-environment.md:56]:
```suggestion
Pod 定义中的用户所定义的环境变量也可在容器中使用，就像在 Docker 镜像中静态指定的任何环境变量一样。
```

File: content/zh/docs/concepts/containers/container-environment.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+Those environment variables match the syntax of Docker links.
+
+For a service named *foo* that maps to a Container named *bar*,
+the following variables are defined:
+-->
+### 集群信息
+
+创建容器时正在运行的所有服务的列表都可用作该容器的环境变量。这些环境变量与 Docker 链接的语法匹配。
+
+对于名为 *foo* 的服务，它映射到名为 *bar* 的容器，定义了以下变量：
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/container-environment.md:71]:
```suggestion
对于名为 *foo* 的服务，当映射到名为 *bar* 的容器时，以下变量是被定义了的：
```

File: content/zh/docs/concepts/containers/container-environment.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+```shell
+FOO_SERVICE_HOST=<the host the service is running on>
+FOO_SERVICE_PORT=<the port the service is running on>
+```
+
+<!--
+Services have dedicated IP addresses and are available to the Container via DNS,
+if [DNS addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/) is enabled. 
+-->
+Service 具有专用的 IP 地址，如果启用了 [DNS插件](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/)，就可以通过 DNS 用于容器。
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/container-environment.md:82]:
```suggestion
Service 具有专用的 IP 地址。如果启用了 [DNS插件](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/)，就可以在容器中通过 DNS 来访问。
```

File: content/zh/docs/concepts/containers/container-environment.md [hunk 7]
```diff
@@ -29,7 +29,7 @@ The Kubernetes Container environment provides several important resources to Con
 
 Kubernetes 的容器环境给容器提供了几个重要的资源：
 
-* 文件系统，包括的一个 [image](/docs/concepts/containers/images/) 和一个或多个的 [volumes](/docs/concepts/storage/volumes/) 的组合。
+* 文件系统，包括的一个 [镜像](/docs/concepts/containers/images/) 和一个或多个的 [卷](/docs/concepts/storage/volumes/) 的组合。
```
Comment (tengqm) [inline content/zh/docs/concepts/containers/container-environment.md:32]: please refer to previous comment. The current translation is not good Chinese.

Comment (tengqm) [reply content/zh/docs/concepts/containers/container-environment.md:32]: 包括的 。。。。 -> 其中包含一个镜像和一个或多个卷。

**Comment by tengqm** [comment]
thanks
/lgtm
/approve

**Pull request merged**